### PR TITLE
Ignore DuplicateBlockProcessing error

### PR DESCRIPTION
**Pull request by vutran1710**

### CODE REVIEW

File: store/postgres/src/deployment_store.rs [hunk 1]
```diff
@@ -1130,9 +1130,19 @@ impl DeploymentStore {
...
             Ok(event)
-        })?;
+        });
 
-        Ok(event)
+        match event {
+            Ok(e) => Ok(e),
+            Err(err) => match err {
+                StoreError::DuplicateBlockProcessing(_deployment_hash, block_number) => {
+                    let default_event = StoreEvent::from_mods(&site.deployment, mods);
```
Comment (ngtuna) [inline store/postgres/src/deployment_store.rs:1139]: what `default_event` will be ?

Comment (vutran1710) [reply store/postgres/src/deployment_store.rs:1139]: Something?

Tracing the stacks upward we can see that, once this function completes without error, no one gonna give a shit about its result.

Note that `site.deployment` & `mods` are immutable inputs as well - meaning they bear no effects to the state of the application either

File: store/postgres/src/writable.rs [hunk 2]
```diff
@@ -571,6 +571,11 @@ impl Queue {
                         // from the queue
                         queue.queue.pop().await;
                     }
+                    Ok(Err(StoreError::DuplicateBlockProcessing(_dh, _bn))) => {
+                        // NOTE: rpc produce duplicate block,
+                        // We expect this already, so just ignore it
```
Comment (halink0803) [inline store/postgres/src/writable.rs:576]: we should add a debug! log here to debug later if needed.

**Review by halink0803 — APPROVED** [review]
LGTM

**Pull request merged**